### DISCUSSION TITLE
墙能翻，但更新有点问题

### DISCUSSION THREAD

**Discussion by sggwqaa** [the1812/Bilibili-Evolved#3404] [提问 / Q&A]
我现在使用的版本是v2.1.9-8-g9c9c5fd27，插件、组件都安装不了，提示错误但没报错信息，我以为是版本已经不被支持了，所以我想更新一下版本，但是自动更新器插件和新版本提示插件始终不显示新版本更新。我试过卸载旧版本，然后在仓库的readme那边点安装，但结果是点击后什么反应都没有，我只能把旧版本装回来。关于readme里提到的两个被墙的网址，我这里一个提示报错500，另一个提示PR_CONNECT_RESET_ERROR，看上去听不正常，但是别的被墙的网站却都能在翻墙后正常浏览（比如谷歌），所以我也不确定我这情况算不算正常。总而言之，我现在不知道该怎么更新了

**Comment by the1812** [comment]
你可能需要检查下翻墙的规则, 确保 README 的两个的网址是通过翻墙而不是直连的
